lib/db/mem.js

```diff
@@ -380,6 +380,27 @@ module.exports = function (log, error) {
       )
   }
 
+  Memory.prototype.sessionDevice = function (id) {
+    id = id.toString('hex')
+
+    if ( !sessionTokens[id] ) {
```

Comment:
nit: extra spacing is not like any other style in this file 

see above ` if (! account.devices[deviceKey]) {` for instance